src/store/applications/sagas.ts

```diff
@@ -153,9 +157,11 @@ export function * requestAPIAccessActionSaga (action: RequestAPIAccessAction) {
     });
 
     yield put(requestAPIAccessSuccess({}));
+    yield put(openNotification("success", t("applications.requestAPIAcessSuccess"), 3000));
```

Comment:
use `i18n.t` instead. That way you don’t force a locale. Check other occurrences.

```suggestion
    yield put(openNotification("success", i18n.t("applications.requestAPIAcessSuccess"), 3000));
```